app/grandchallenge/reader_studies/views.py

```diff
@@ -1410,13 +1429,20 @@ def new_interface_url(self):
 
     def process_data_for_object(self, data):
         """Creates a display set"""
-        instance = DisplaySet.objects.create(reader_study=self.base_object)
-        instance.order = data.pop("order")
-        instance.save()
-        for slug in data:
+        non_civ_data_keys = ("order", "title")
```

Comment:
This is repeated earlier. Might be good to refactor so we only need to change this in one place in the future.

Reply:
Good point, I opted to make the retrieval of these fields dynamic in that it's the existing fields minus the interface fields.